diff -urN dolphin-emu-3.0.rev20120321/CMakeLists.txt dolphin-emu-3.0.rev20120321-patched/CMakeLists.txt
--- dolphin-emu-3.0.rev20120321/CMakeLists.txt	2012-03-21 12:18:48.000000000 +1100
+++ dolphin-emu-3.0.rev20120321-patched/CMakeLists.txt	2012-03-21 12:32:33.798271511 +1100
@@ -25,7 +25,7 @@
 else()
 	set(bindir  ${CMAKE_INSTALL_PREFIX}/bin               CACHE PATH "bindir")
 	set(datadir ${CMAKE_INSTALL_PREFIX}/share/dolphin-emu CACHE PATH "datadir")
-	add_definitions(-DDATA_DIR="${datadir}/")
+	add_definitions(-DDATA_DIR="/usr/share/dolphin-emu/")
 endif()
 set(userdir ".dolphin-emu" CACHE STRING "User directory")
 add_definitions(-DUSER_DIR="${userdir}")
